### ISSUE TITLE
docset can not break line

### ISSUE THREAD

**Issue by callmewhy** [Kapeli/cheatset#22]
It seems that there is a style error in the generated docset.

The .rb file:

```
  category do
    id 'Functor'
    entry do
      notes <<-'END'
        ```swift
        1
        2
        3
        ```
      END
    end
  end
```

generated docset:
```
1 2 3 
```

![2016-06-28 at 19 10](https://cloud.githubusercontent.com/assets/6276527/16413352/027dbb36-3d64-11e6-955f-920e68eb7048.png)

**Comment by Kapeli** [comment]
This seems to work fine for me.

From this:

![screen shot 2016-06-28 at 16 33 27](https://cloud.githubusercontent.com/assets/1445635/16417347/209db3f2-3d4e-11e6-95c5-7058b9c3f101.png)

I get this:

![screen shot 2016-06-28 at 16 33 44](https://cloud.githubusercontent.com/assets/1445635/16417352/25841d20-3d4e-11e6-9af1-79b00425dd44.png)

**Comment by callmewhy** [comment]
Maybe something wrong after updating all dependencies?
For me:

```
cheatset (1.3.3)
rouge (2.0.2)
redcarpet (3.3.4)
```

**Comment by Kapeli** [comment]
I was able to replicate the issue and fix it after updating to the latest version of rouge. I've released a new version of cheatset (1.3.4)
